compiler/optimizer/RegisterCandidate.hpp

```diff
@@ -63,25 +62,13 @@ class GlobalSet
    bool isEmpty() { return _refAutosPerBlock.empty(); }
    void makeEmpty() { _refAutosPerBlock.clear(); }
 
-   class Set
-      {
-      public:
-      TR_ALLOC(TR_Memory::RegisterCandidates);
-      Set(TR::Region &region):_refs(region){}
-      bool get(uint32_t refId) { return _refs.get(refId); }
-      void set(uint32_t refId) { _refs.set(refId); }
-      void print(TR::Compilation * comp);
-      private:
-      TR_BitVector _refs;
-      };
-
```

Comment:
Originally this abstraction served to wrap two different BitVector implementations using the CS2 sparse and dense bit vectors. These bit vectors were change to TR_BitVectors for memory savings and compatibility with region memory allocation. That rendered the hierarchy meaningless and a memory consumer hence why it is removed in my commits.